.github/CONTRIBUTING.md

```diff
@@ -23,3 +25,39 @@ You can report bugs or make enhancement requests at the [Ansible GitHub issue pa
 Also please make sure you are testing on the latest released version of Ansible or the development branch; see the [Installation Guide](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) for details.
 
 Thanks!
+
+## DEVELOPER CERTIFICATE OF ORIGIN (DCO) VERSION 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
```

Comment:
Should the DCO be enclosed in a `text` code block to preserve the original formatting? Currently, some of the original plain text formatting is lost -- such as the first four lines being rendered as one.

Reply:
Yeah, that probably makes sense, thanks.